src/main/java/nextstep/courses/domain/ImageType.java

```diff
@@ -0,0 +1,25 @@
+package nextstep.courses.domain;
+
+public enum ImageType {
+    GIF,
+    JPG,
+    JPEG,
+    PNG,
+    SVG;
+
+    public static ImageType typeCheck(String imageType) {
+        validImageTypeIsNull(imageType);
+        try {
+            return ImageType.valueOf(imageType);
+        } catch(IllegalArgumentException e) {
+            throw new IllegalArgumentException("지원하지 않는 이미지 타입");
+        }
```

Comment:
Nit: Java의 stream을 활용해보셔도 좋을 것 같습니다.
```suggestion
        return Arrays.stream(values())
            .filter(type -> type.name().equalsIgnoreCase(imageType))
            .findFirst()
            .orElseThrow(() -> new IllegalArgumentException("지원하지 않는 이미지 타입"));
```